inc/template-tags.php

```diff
@@ -26,6 +26,8 @@
  *
  * @param array   $args Arguments for displaying the site logo either as an image or text.
  * @param boolean $echo Echo or return the html.
+ *
+ * @return string $html Compiled html based on our arguments.
```

Comment:
```suggestion
 * @return string $html Compiled HTML based on our arguments.
```

Reply:
Uppercase HTML for consistency with the other instances of this in this PR

Reply:
@itowhid06 In line 28 there's another `html` which with should be `HTML`.